.gitlab/functional_test/regression_detector.yml

```
@@ -136,3 +133,43 @@ single-machine-performance-regression_detector:
     - RUST_LOG="${RUST_LOG}" ./smp --team-id ${SMP_AGENT_TEAM_ID} --api-base ${SMP_API} --aws-named-profile ${AWS_NAMED_PROFILE}
       job result
       --submission-metadata submission_metadata
+
+# Shamelessly adapted from golang_deps_commenter job config in
+# golang_deps_diff.yml at commit 01da274032e510d617161cf4e264a53292f44e55.
+single-machine-performance-regression_detector-commenter:
+  stage: functional_test
+  rules:
+    - !reference [.except_main_or_release_branch]
```

```suggestion
    - !reference [.except_main_or_release_branch]
    - when: on_success
```
This is mandatory for the job to be generated, otherwise it matches the `!reference` that has only exclusion rules (so nothing requires the job generation)

---

Added in https://github.com/DataDog/datadog-agent/pull/29785/commits/49a0419691ff29dd4efe3244397f4bde20ada908, in #29785.